grub2-efi/install

```diff
@@ -11,17 +11,23 @@ case "$target" in
   i?86 ) target=i386 ;;
   x86_64 | amd64 ) target=x86_64 ;;
   aarch64 ) target=arm64 ;;
-  armv7l ) target=arm ;;
+  arm* ) target=arm ;;
 esac
 
 target="$target-efi"
 
 echo "target = $target"
 
+# Systems without efivars should just fall back to install grub as removable
```

Comment:
U-Boot can have configuration space, but it's either stored raw on Flash or in a text file on some media (SD Card for example). Both flash and SD card are not guaranteed to be owned by U-Boot once Linux is up though. So we would have 2 different pieces of software colliding on device access (which means hell breaks loose).

So the short answer is that U-Boot can not write its environment once Linux is up :).